curves/src/traits/group.rs

```diff
@@ -47,6 +48,7 @@ pub trait Group:
     + AddAssign<Self>
     + SubAssign<Self>
     + MulAssign<Self::ScalarField>
+    + Sum
```

Comment:
Hm I think we should add this trait bound to the `ProjectiveCurve` trait, instead of here. The `Group` trait should probably be deprecated and removed.

Reply:
You got it :ok_hand:.